nim/happyx/config.yaml

```diff
@@ -1,9 +1,6 @@
 framework:
   github: HapticX/happyx
-  version: "0.26"
+  version: "1.0"
 
-deps:
-  - pcre-dev
-
-build_opts: '--checks:off --assertions:off --warnings:off --hints:off -d:httpx --define:usePcreHeader --passL: "/usr/lib/libpcre.a"'
+build_opts: '--checks:off --assertions:off --warnings:off --hints:off -d:httpx --threads:on --mm:orc'
```

Comment:
I see that you use `orc`, is it not something that we could recommend to all nim frameworks ?

Also `--checks:off --assertions:off --warnings:off --hints:off --threads:on` seems to be generic, should not we use the same compilation option for nim frameworks ?

Reply:
threads and ORC is not universal, some frameworks can be not support these features

Reply:
Ok, then

Is this the same for `--checks:off --assertions:off --warnings:off --hints:off` ?

Reply:
no, these flags is universal
`hints:off` just removes compile-time hints
`checks`, `assertions` and `warnings` removes runtime code

Reply:
@waghanza please see a new commits

Reply:
@bung87 compilation error `https://github.com/the-benchmarker/web-frameworks/actions/runs/5164257080/jobs/9303043497?pr=6409#step:7:153` with 
```nim
 nim c  \
    --excessiveStackTrace:off \
    --assertions:off \
    --checks:off \
    --warnings:off \
    --hints:off \
    -d:release \
    --opt:speed \
    --passC:-flto \
    --passL:-flto \
    server.nim
```

Reply:
> /root/.nimble/pkgs/scorper-1.1.4/scorper/http/streamserver.nim(392, 26) Error: ambiguous identifier: 'Time' -- use one of the following:

This not related to `assertions`, `hints`, `warnings` or `checks`, but I try to remove `checks` and `assertions` from dockerfile

Reply:
@waghanza 
please see a new commits

Reply:
@Ethosa please restore them, we will look about `scorper` issue in an other PR

Reply:
> @bung87 compilation error `https://github.com/the-benchmarker/web-frameworks/actions/runs/5164257080/jobs/9303043497?pr=6409#step:7:153` with
> 
> ```nim
>  nim c  \
>     --excessiveStackTrace:off \
>     --assertions:off \
>     --checks:off \
>     --warnings:off \
>     --hints:off \
>     -d:release \
>     --opt:speed \
>     --passC:-flto \
>     --passL:-flto \
>     server.nim
> ```

don't know what causes, now I patched to v1.1.4 using fully qualified name, runs fine on local.

Reply:
> don't know what causes, now I patched to v1.1.4 using fully qualified name, runs fine on local.

@bung87 https://github.com/the-benchmarker/web-frameworks/actions/runs/5164421290/jobs/9303307552#step:7:85
![image](https://github.com/the-benchmarker/web-frameworks/assets/49402667/32f08861-3735-4911-89f1-bdc6aec50131)

Reply:
![image](https://github.com/the-benchmarker/web-frameworks/assets/49402667/177ccc87-d6f6-4394-a670-672b37655b4e)

Reply:
I saw that, but it runs on stable and devel without any modification on my m1 MBP.

Reply:
seems that ecdc67bbd00ebfe2a5d7d7287a416ba31d661f5b fixes things on https://github.com/bung87/scorper/commits/v1.1.4 @bung87 :-)